candlepin.spec.tmpl

```diff
@@ -75,7 +75,15 @@ Summary:        SELinux policy module supporting candlepin
 Group:          System Environment/Base
 BuildRequires:  checkpolicy
 BuildRequires:  selinux-policy-devel
-BuildRequires:  hardlink
+%if (0%{?rhel} && 0%{?rhel} >= 9) || (0%{?fedora} && 0%{?fedora} >= 35)
+BuildRequires: util-linux-core
+%endif
+%if (0%{?fedora} && 0%{?fedora} >= 29 && 0%{?fedora} < 35)
+BuildRequires: util-linux
+%endif
+%if (0%{?rhel} && 0%{?rhel} < 9) || (0%{?fedora} && 0%{?fedora} < 29)
```

Comment:
any Fedora < 34 is EOL, so should older Fedora versions still be supported?

Reply:
good point, I'll remove this

Reply:
done